customStartup_client.sqf

```diff
@@ -0,0 +1,8 @@
+// Use this file to run code when the mission starts, or a player joins.
+
+// This file runs client-side, so code here will run for every player.  If there are 20 players, it will run 20 times (1 each)!
```

Comment:
I understand where this is coming from, but I don't fully agree with the wording - let me explain:

The file is executed once *on every machine*. This means, the client's machine executes it *once* - irrespective of how many players are on the server.

The comment describes a special case: the usage of code with global effect. In that case, the global effect will be reapplied for every connected (and connecting) player. However, I don't think it bears mentioning, because this file is intended for clientside usage only (which inherently forbids anything with global effect). Anything else should go into the *serverside* init file (that's why it exists).

**tl;dr:** valid point, but misleading phrasing.

Reply:
I want a warning for this kind of stuff because locality is a thing even I'm still screwing up all the time, and MMs aren't always programmers.  But yeah I'm not wedded to the wording, if anyone's got clarity suggestions we can go with those.